templates/envs/goerli/execution/erigon.tmpl

```diff
@@ -0,0 +1,8 @@
+{{/* erigon.tmpl */}}
+{{ define "execution" }}
+# --- Execution Layer - Execution Node - configuration ---
+EC_IMAGE_VERSION={{if .ElImage}}{{.ElImage}}{{else}}thorax/erigon:devel{{end}}
```

Comment:
Think we should use a specific tested version so we don't have trouble with future erigon releases. Even more if the default ones its a development branch.